### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_NBExtensionsTab_WAT: Color Contrast is less than 4.5:1 for Reset Link.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4002]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106 
Operating System:Windows10
 
 **Steps to reproduce:**
1. Login with the appropriate credentials to azure notebook account.    
2. Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3. Click on Run or right click and select Run to run the notebook.
4. Navigate to NBExtensionTab.
5. Navigate till "Parameters" Dropdown present on NBExtensionTab. 
6. Open WAT Tool.
7. Open Color Contrast Analyser.
8. Verify Color Contrast for Reset Link.

**Actual:**
Color Contrast is less than 4.5:1 for Reset Link.

**Expected:**
Color Contrast should be >= 4.5:1 for Reset Link.

**User Impact:**
If ColorContrast is <4.5:1 than Low Vision Users face difficulty to See the Links.

**Attachment:**
![a11y_azurenotebooks_jupyter_nbextensionstab_wat](https://user-images.githubusercontent.com/39204909/45815472-65c22880-bcf6-11e8-90bb-fcb71dbbbcfb.png)

**Comment by takluyver** [comment]
Fixed by #4679
